lstchain/datachecks/containers.py

```diff
@@ -129,9 +129,9 @@ def fill_event_wise_info(self, subrun_index, table, mask, geom,
         n_jump = 1+int(self.num_events/n_samples)
         # keep some info every n-jump-th event:
         sampled_event_ids = np.array(table['event_id'][mask][0::n_jump])
-        tib_time = np.array(table['tib_time'][mask][0::n_jump])
-        ucts_time = np.array(table['ucts_time'][mask][0::n_jump])
-        dragon_time = np.array(table['dragon_time'][mask][0::n_jump])
+        tib_time = np.array(table['tib_time'][mask][0::n_jump])*u.s
```

Comment:
Do not use `array * u.<unit>`, it is much slower than `u.Quantity(somearray, u.unit, copy=False)`

Reply:
ok!